cpp/example_code/sns/README.md

```diff
@@ -1,4 +1,11 @@
-<!--Generated by WRITEME on 2023-05-15 12:45:10.155329 (UTC)-->
+### Scenarios
```

Comment:
This looks misplaced.

Reply:
Yeah, I copied and pasted because of WRITEME does not handle it correctly. I will remove it.